src/Components/Icon/UserIcon.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Icon } from 'semantic-ui-react';
+import usericon from './userIcon.css'
```

Comment:
@Justin-ml Why do we need `usericon` for CSS imports?

Reply:
removed that part.